src/components/FoldedListbox/SelectionSegmentsIndicator.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { Grid } from '@mui/material';
+import getSegmentsSizes from './get-segment-sizes';
+
+interface SelectionSegmentsIndicatorProps {
```

Comment:
`ISelectionSegmentsIndicatorProps`?

Reply:
Isn't it enough with the `props` suffix? (Both `i` and `prop` is a bit cookie on cookie as you say in sweden? :))